sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvalHelper.scala

```diff
@@ -57,8 +57,8 @@ trait ExpressionEvalHelper extends GeneratorDrivenPropertyChecks {
     (result, expected) match {
       case (result: Array[Byte], expected: Array[Byte]) =>
         java.util.Arrays.equals(result, expected)
-      case (result: Double, expected: Spread[Double]) =>
-        expected.isWithin(result)
+      case (result: Double, expected: Spread[_]) => // Can't use Spread[Double] b/c of erasure
```

Comment:
@thunterdb  Hm, I get this error now:

```
[error] /home/jenkins/workspace/SparkPullRequestBuilder/extras/kinesis-asl/src/test/scala/org/apache/spark/streaming/kinesis/KinesisStreamSuite.scala:140: No Manifest available for org.apache.spark.streaming.kinesis.KinesisBackedBlockRDD[Array[Byte] @unchecked].
[error]     nonEmptyRDD shouldBe a [KinesisBackedBlockRDD[Array[Byte] @unchecked]]
[error]      
```

Did I misunderstand how to apply it or is it not going to work here?